android/app/build.gradle

```diff
@@ -210,6 +210,11 @@ dependencies {
     implementation('com.crashlytics.sdk.android:crashlytics:2.9.5@aar') {
         transitive = true
     }
+    implementation ('org.jitsi.react:jitsi-meet-sdk:2.1.0', {
```

Comment:
we have a 2.2.2 already which solves a number of bugs, I'd suggest you update.

Reply:
Nice! ❤️